src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -52,18 +70,25 @@ end
     LSLogMA <: LinSysSolverLog
 
 A mutable structure that stores information about a randomized linear solver's behavior.
-    The log assumes that only a random block of full linear system is available for processing. 
-    The goal of this log is usually for all use cases as it acts as a cheap approximation of the 
-    full residual.
+The log assumes that only a random block of full linear system is available for processing. 
+The goal of this log is usually for all use cases as it acts as a cheap approximation of the 
+full residual.
+
+For more information see:
+Pritchard, Nathaniel, and Patel, Vivak. Solving, tracking and stopping streaming linear inverse problems. 
+Inverse Problems 40.8 Web. doi:10.1088/1361-6420/ad5583.
+
+Pritchard, Nathaniel and Vivak Patel. “Towards Practical Large-Scale Randomized Iterative Least Squares 
+Solvers through Uncertainty Quantification.” SIAM/ASA J. Uncertain. Quantification 11 (2022): 996-1024.
 
 # Fields
 - `collection_rate::Int64`, the frequency with which to record information to append to the
-    remaining fields, starting with the initialization (i.e., iteration 0).
+    remaining fields, starting with the initialization (i.e., iteration `0`).
 - `ma_info::MAInfo`, structure that holds information relevant only to moving average, 
-   like the two choices of moving average widths (lambda1 and lambda2), the current moving average
-   width (lambda), a flag to indicate which moving average regime, between lambda1 and lambda2, is 
-   being used, a vector containing the values to be averaged (res_window), and an indicator of where 
-   the next value should be placed (idx).
+   like the two choices of moving average widths (`lambda1` and `lambda2`), the current moving average
+   width (`lambda`), a flag to indicate which moving average regime, between `lambda1` and `lambda2`, is 
+   being used, a vector containing the values to be averaged (`res_window`), and an indicator of where 
+   the next value should be placed (`idx`).
 - `resid_hist::Vector{Float64}`, a structure that contains the moving average of the error proxy, 
```

Comment:
A vector that contains the moving average estimate of the error proxy which is collected at the collection rate specified by `collection_rate`.

Reply:
I have made this change